cmd/minikube/cmd/image.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/minikube/pkg/minikube/config"
+	"k8s.io/minikube/pkg/minikube/exit"
+	"k8s.io/minikube/pkg/minikube/machine"
+	"k8s.io/minikube/pkg/minikube/reason"
+)
+
+// imageCmd represents the image command
+var imageCmd = &cobra.Command{
```

Comment:
It would be simpler if "load" was a top-level command, like all other minikube commands

Reply:
Just commented this on the issue as well:

>This could be extended into minikube image build or minikube image delete. I agree that right now just minikube load would be easier but for future it could be nice to organize all image related content under one subcommand.

Reply:
@afbjorklund the reason for adding "image load" is we need other commands 
```
image delete
image list
image build
```

without a sub command it woudl be hard to add other operations for image
we could add a "hidden" alias "load"